master/usage/assigning-ip-addresses-topology.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Assigning IP addresses based on topology
+---
+
+## Why this is useful
+
+{{site.prodname}} can be configured to use specific IP pools for different topological areas.
+For example, you may want workloads in a particular rack, zone, or region to receive addresses from the same IP pool.
+This may be desirable either to reduce the number of routes required in the network or to meet requirements imposed by an external firewall device or policy.
+This guide walks through an example of how to set this up.
+
+## How this works
+At a high level, this feature is operated by the setting node labels and then selecting those
+nodes via node selector on your IP pool resources. Note that there are a variety of other ways
+to specify IP address assignment behavior that can be found in the
+[cni-plugin configuration reference document]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration).
+Each approach has its own use-case but this particular one takes on the lowest priority
+compared to the others (i.e. Specifying IP pools through configuration has precendence
+over IP addresses through annotations which has precendence over this IP pool
+assignment through node selectors).
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+### Example: Kubernetes
+
+In this example, we created a cluster with four nodes across two racks (two nodes/rack). Consider the following:
+
+```
+       -------------------
+       |    router       |
+       -------------------
+       |                 |
+---------------   ---------------
+| rack-0      |   | rack-1      |
+---------------   ---------------
+| kube-node-0 |   | kube-node-2 |
+- - - - - - - -   - - - - - - - -
+| kube-node-1 |   | kube-node-3 |
+- - - - - - - -   - - - - - - - -
+```
+
+Using the pods IP range `192.168.0.0/16`, we target the following setup: reserve the `192.168.0.0/24` and `192.168.1.0/24`
```

Comment:
I believe that pods here is a possessive. As such, it should have an apostrophe. So: `s/pods/pod's`

Reply:
there should actually be no `s` at all